esque/cli/commands.py

```diff
@@ -33,11 +33,15 @@
 from esque.resources.topic import Topic, copy_to_local
 
 
-@click.group(help="esque - an operational kafka tool.", invoke_without_command=True)
+@click.group(invoke_without_command=True)
 @click.option("--recreate-config", is_flag=True, default=False, help="Overwrites the config with the sample config.")
 @version_option(__version__)
 @default_options
 def esque(state: State, recreate_config: bool):
+    """esque - an operational kafka tool.
```

Comment:
Can we make it possible to show the help when `esque` is called without any options or arguments? Currently nothing is shown.

Reply:
Yes!